db/migrate/20180315164256_add_tracker_type_to_spree_trackers.rb

```diff
@@ -0,0 +1,9 @@
+class AddTrackerTypeToSpreeTrackers < SolidusSupport::Migration[4.2][5.0]
+  def up
+    add_column :spree_trackers, :tracker_type, :string
```

Comment:
I'd still add a default to analytics here or people will have to update all the old values manually. Also, why not just `type`? Is it an issue with STI?

Reply:
yes, I named `tracker_type` because of STI, I added the default value here https://github.com/solidusio-contrib/solidus_trackers/pull/10/commits/3e91f85f6d7ad0b240680e8ebf7d81edcb72795e#diff-a4bcd4674f5cedb785641cd75e7dae09R3